frontend/src/tests/session-tests.js

```diff
@@ -76,12 +78,12 @@ export function sessionsTests(api) {
         expect(withNewSessions.map(x => x.id)).toContain(createdSession.id);
 
```

Comment:
Should also check that `expect(withNewSessions).toContainObject(createdSession)`? Like what we did in `verifySeededDatabase`

Reply:
sure